### PR TITLE
chore: Sweep CI

### PR DESCRIPTION
**Pull request by Hoverbear**
Attempt to clean some disk space on our CI.

### CODE REVIEW

**Comment by jamtur01** [comment]
Many failed jobs make CI sad. :(

**Comment by Hoverbear** [comment]
I forgot a `?`.

**Comment by Hoverbear** [comment]
@MOZGIII gave me some fine CI VM butchery tips.

File: Makefile [hunk 1]
```diff
@@ -977,6 +980,24 @@ signoff: ## Signsoff all previous commits since branch creation
...
+	df -h
+	sudo apt-get install -y aptitude ubuntu-minimal
+	sudo aptitude markauto -y '~i!~M!~prequired!~pimportant!~R~prequired!~R~R~prequired!~R~pimportant!~R~R~pimportant!busybox!grub!initramfs-tools'
+	sudo aptitude hold -y moby-engine moby-cli
+	sudo aptitude purge -y '~c'
+	sudo apt-get --purge autoremove
+	sudo apt-get clean
+	sudo rm -rf "/opt/*" "/usr/local/*"
+	sudo rm -rf "/usr/local/share/boost" && sudo rm -rf "${AGENT_TOOLSDIRECTORY}"
+	mapfile -t DOCKER_IMAGES < <(docker image ls -q)
```
Comment (jszwedko) [inline Makefile:996]: TIL `mapfile`

Comment (Hoverbear) [reply Makefile:996]: @MOZGIII magics

**Review by jamtur01 — APPROVED** [review]
This is LGTM

**Comment by MOZGIII** [comment]
A bit late, but my autoremove magic only works to full power if all the unwanted packages are marked as autoinstalled (those `sudo aptitude ...` lines).

**Comment by jszwedko** [comment]
@Hoverbear I might be missing something, but this doesn't actually seem to make any additional room in `/mnt` which, based on the 14 GB number listed on https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners, appears to be the filesystem used by builds? Am I understanding that correctly?

**Comment by jszwedko** [comment]
It seems to have worked though! :smile: 

https://github.com/timberio/vector/runs/1189133020?check_suite_focus=true

**Pull request merged**